runtime/doc/options.txt

```diff
@@ -3852,7 +3852,18 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Note that using the "-u NONE" and "--noplugin" command line arguments
 	reset this option. |-u| |--noplugin|
 
-						*'magic'* *'nomagic'*
+					*'luaintchkfreq'* *'licf'*
+'luaintchkfreq' 'licf'	number	(default 100)
+			global
+	Determines how often (in lua opcodes) lua bindings should check for
+	incoming interrupts: with default value 100 it will check for
+	interrupt after running 100 opcodes.  The lesser the number, the
+	slower lua code runs.  The special value of 0 prevents lua from
+	checking for interrupts completely, but allows using `debug.sethook` for
+	other purposes: with interrupts checking enabled it is being used for
+	checking for interrupts.
```

Comment:
1. 'luaintchkfreq' is a confusing name because "frequency" implies "checks per time", but this is "opcodes per check".
2. Some sort of concrete example will help users here.

```
Simple loops produce very few opcodes, so ctrl-c will not interrupt such
a loop unless the check frequency is very high. For example the
following loop produces only 3 opcodes:

  local i = 0
  while true do
    i = i + 1
  end
```